.github/workflows/ci-main-daily.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: authentik-ci-main-daily
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every night at 3am
+    - cron: "0 3 * * *"
+
+jobs:
+  test-container:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - docs
+          - version-2024-12
```

Comment:
We should probably add to the release checklist that this needs to be updated as well

Reply:
Yeah I have some plans for automating and centralising that